includes/polling/applications/ups-nut.inc.php

```diff
@@ -29,7 +29,7 @@
 $name = 'ups-nut';
 $app_id = $app['app_id'];
 $oid = '.1.3.6.1.4.1.8072.1.3.2.3.1.2.7.117.112.115.45.110.117.116';
-$ups_nut = snmp_get($device, $oid, '-Oqv');
+$ups_nut = trim(snmp_get($device, $oid, '-Oqv'), '"');
```

Comment:
We already trim `"` : https://github.com/librenms/librenms/blob/master/includes/snmp.inc.php#L203

Are you running the latest version?

Reply:
root@speedtest:/opt/librenms# ./validate.php
====================================
Component | Version
--------- | -------
LibreNMS  | 1.32-23-gf03cf599f
DB Schema | 210
PHP       | 7.0.19-1
MySQL     | 10.1.26-MariaDB-0+deb9u1
RRDTool   | 1.6.0
SNMP      | NET-SNMP 5.7.3
====================================

[OK]    Database connection successful
[OK]    Database schema correct
[WARN]  Your local git branch is not master, this will prevent automatic updates.